### ISSUE TITLE
linter-flow does not lint files without `@flow` annotation even when "Enable all" is enabled

### ISSUE THREAD

**Issue by gnestor** [AtomLinter/linter-flow#105]
To reproduce: 

* Create a JS file with the following:

```js
/* @flow */
function bar(x: string, y: number): string {
  return x.length * y
}
bar('Hello', 42)
```

* "Enable all"
* Remove the `/* @flow */` annotation
* Linter does not catch the type error

**Comment by Arcanemagus** [comment]
What exactly do you mean by `"Enable all"`?

**Comment by gnestor** [comment]
"Enable all" is one of the 2 config options: https://github.com/AtomLinter/linter-flow/blob/39603f4b1003bddb600883c6de19e1a4a53a7d33/README.md#L24

However, I looked at the source and it looks like it's not implemented...

**Comment by Arcanemagus** [comment]
Yea, it looks like that option was removed at some point and the readme was just never updated, trying to find where...

**Comment by Arcanemagus** [comment]
It [looks like](https://github.com/AtomLinter/linter-flow/pull/57#issuecomment-196670773) that option was removed since `flow` versions past v0.22 no longer implement an option that would enable that to work.

**Comment by Arcanemagus** [comment]
Btw, to make that option no longer show up simply remove it from your `config.cson`, for whatever reason Atom shows _all_ options in the settings, regardless of whether the package implements them or not.

**Comment by gnestor** [comment]
Ok thanks @Arcanemagus
